src/VirtoCommerce.NotificationsModule.Data/Repositories/NotificationRepository.cs

```diff
@@ -74,5 +67,27 @@ public async Task<NotificationMessageEntity[]> GetMessagesByIdsAsync(string[] id
             }
             return result;
         }
+
+
+        private async Task GetNestedEntitiesAsync(string[] ids, string responseGroup)
```

Comment:
Make protected and virtual. And rename to `LoadNotifcationDependenciesAsync`

Reply:
sure, right!